### PR TITLE
UrlScan: Update pack metadata and README

### PR DESCRIPTION
**Pull request by heipei**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/UrlScan/README.md [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
+This includes the domains and IPs contacted, the resources (JavaScript, CSS,
+etc) requested from those domains, file downloads that were triggered, as well
+as additional information about the page itself. urlscan.io will take a
+screenshot of the page, record the DOM content, JavaScript global variables,
+cookies created by the page, and a myriad of other observations. If the site is
+targeting the users one of the more than 660 brands tracked by urlscan.io, it
+will be highlighted as potentially malicious in the scan results.
 
 To use the urlscan.io integration, sign up for a free user account and create
 an API key first: https://urlscan.io/user/signup
```
Comment (mchasepan) [inline Packs/UrlScan/README.md:16]: Suggest you make these links

File: Packs/UrlScan/README.md [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
 
+urlscan.io has a free user tier which will allow you to perform a little number
+of scans and searches on our platform. If you need to perform more scans you
+can subscribe to one of our commercial API plans available on
+https://urlscan.io/products/. If you are also interested in a Threat
```
Comment (mchasepan) [inline Packs/UrlScan/README.md:21]: again, links

File: Packs/UrlScan/README.md [hunk 3]
```diff
@@ -1,22 +1,27 @@
...
+as additional information about the page itself. urlscan.io will take a
+screenshot of the page, record the DOM content, JavaScript global variables,
+cookies created by the page, and a myriad of other observations. If the site is
+targeting the users one of the more than 660 brands tracked by urlscan.io, it
+will be highlighted as potentially malicious in the scan results.
 
 To use the urlscan.io integration, sign up for a free user account and create
 an API key first: https://urlscan.io/user/signup
 
+urlscan.io has a free user tier which will allow you to perform a little number
```
Comment (mchasepan) [inline Packs/UrlScan/README.md:18]: "little number"? Can you detail the offering more?

**Comment by mchasepan** [comment]
@marcellarichmond FYI

**Comment by heipei** [comment]
I have pushed an additional commit to this branch to address the comments by @mchasepan

**Review by tallieber — APPROVED** [review]

**Comment by tallieber** [comment]
Hey @heipei, as part of pack adoption you need to add also a release notes indicates it, see details [here](https://xsoar.pan.dev/docs/partners/adopt#process) 
Also, there is a conflict here, you need to merge from master and fix the conflict, afterwards I will approve and merge.

**Comment by heipei** [comment]
I forced-pushed this branch again, rebase against current master.

I don't know how to create release notes, the demisto-sdk command comes up with:
```You are using demisto-sdk 1.5.0.
Starting to update release notes.
Either no changes were found in UrlScan pack or the changes found should not be documented in the release notes file.
If relevant changes were made, please commit the changes and rerun the command.
```

I also can't see the merge conflicts since the LGTM tool seems to be broken. I didn't make any changes to Python or JS files, so I don't know how my commit could have caused a merge commit there.

**Comment by tallieber** [comment]
> I forced-pushed this branch again, rebase against current master.
> 
> I don't know how to create release notes, the demisto-sdk command comes up with:
> 
> ```
> Starting to update release notes.
> Either no changes were found in UrlScan pack or the changes found should not be documented in the release notes file.
> If relevant changes were made, please commit the changes and rerun the command.
> ```
> 
> I also can't see the merge conflicts since the LGTM tool seems to be broken. I didn't make any changes to Python or JS files, so I don't know how my commit could have caused a merge commit there.

You can use the force flag e.g. ` demisto-sdk update-release-notes -f -i <pack_path>` so that release notes will create and you can write about the pack adoption as explained [here](https://xsoar.pan.dev/docs/partners/adopt#process)

regarding the conflict, this happens on the pack_metadata file as the current version is already 1.1.11. so maybe you dont use the current version of the pack.

**Comment by heipei** [comment]
Updated branch with Release Notes pushed

**Comment by tallieber** [comment]
> Updated branch with Release Notes pushed

This is the end of the 90 day period? There is a format if you please use him:

`Note: Support for this pack moved to the partner on <<Month>>, <<Day>>, <<Year>>. Please contact the partner directly via the support link on the right.`

**Comment by heipei** [comment]
The CircleCI build fails because of a missing Author.yml, but I don't see that file mentioned anywhere in the guides, and I don't see any other pack using that file.

**Comment by tallieber** [comment]
> The CircleCI build fails because of a missing Author.yml, but I don't see that file mentioned anywhere in the guides, and I don't see any other pack using that file.

Yes you are right. We have a bug in our validation. This should be fix today, I will take care of this PR from now. 
Thank you!

**Review by tallieber — APPROVED** [review]

**Pull request merged**